index.js

```diff
@@ -1,16 +1,67 @@
-var BigNumber = require('bignumber.js')
-
 var Units = {}
 
 var rawUnits = require('./units.json')
 var units = {}
 
 Object.keys(rawUnits).map(function (unit) {
-  units[unit] = new BigNumber(rawUnits[unit], 10)
+  // drop the leading number 1
+  units[unit] = rawUnits[unit].length - 1
 })
 
 Units.units = rawUnits
 
+// add leading zeroes
+// FIXME: improve
+function zeroes (n) {
```

Comment:
> I was thinking about this, zeroes would need to be either 31 zeroes (256bits) or the largest in units.json. Any quick way to find the largest unit in units.json?

Sorry, not familiar with ethereum units.. maybe 256-characters length string will be enough? :)